synthesis/synthesis.go

```diff
@@ -162,94 +154,23 @@ func findProblems(sequence string, problematicSequenceFuncs []func(string, chan
 // FixCds fixes a CDS given the CDS sequence, a codon table, a list of
 // functions to solve for, and a number of iterations to attempt fixing.
 // 10 iterations is a reasonable default for fixIterations.
-func FixCds(sqlitePath string, sequence string, codontable codon.Table, problematicSequenceFuncs []func(string, chan DnaSuggestion, *sync.WaitGroup), fixIterations int) (string, []Change, error) {
+func FixCds(sequence string, codontable codon.Table, problematicSequenceFuncs []func(string, chan DnaSuggestion, *sync.WaitGroup), fixIterations int) (string, []Change, error) {
```

Comment:
Should include some reference to FixCds simple for users that don't want to muck with details. Will save us time answering questions in long run.

Reply:
Yea, [it'd be a shame if some users](https://github.com/TimothyStiles/poly/blob/cfe26e05c3cac061452211ecccf14427230f397b/synthesis/synthesis.go#L10-L12) didn't read the docs. Will add.